binaries/ofs.uio/uio/ofs_uio.py

```diff
@@ -96,19 +99,20 @@ def verify_json(file_json):
         try:
             data = json.load(f)
         except ValueError:
-            print('Decoding JSON has failed')
+            LOG.error('Decoding JSON has failed')
             raise argparse.ArgumentTypeError('Decoding JSON has failed', file_json)
 
         for i in data:
             if i.get('address') is not None:
                 if verify_hex(i.get('address')) is False:
                     raise argparse.ArgumentTypeError('{} file contains invalid hex address {} {}'
-                    .format(file_json, i.get('reg-name'), i.get('address')))
+                                                     .format(file_json, i.get('reg-name'),
```

Comment:
Thanks for the updates.  I think there is one straggler here.

Reply:
format used in ArgumentTypeError() function, not in LOG.debug() or LOG.info

Reply:
I see, my mistake.